Mathlib/GroupTheory/Subgroup/MulOpposite.lean

```diff
@@ -17,6 +17,52 @@ subgroup, subgroups
 
 
 variable {G : Type*} [Group G]
+variable {M : Type*} [MulOneClass M]
+
+namespace Submonoid
```

Comment:
I think this should go in a new `Mathlib/GroupTheory/Submonoid/MulOpposite.lean` file.

Reply:
(this might be a job for a spin-off PR)

Reply:
done